Add docs for insights (#17053)
dagster-io/dagster: docs/content/dagster-cloud/insights.mdx

---
title: Dagster Cloud Insights
description: "Visibility into historical usage and cost metrics."

platform_type: "cloud"
---

# Dagster Cloud Insights

<Note>
  This feature is considered <strong>experimental</strong>. To get access to
  Insights please request access in the
  <a href="https://dagster.slack.com/archives/C05V7GETFSQ">#dagster-insights</a>
  channel in Slack.
</Note>

Insights is Dagster Cloud feature that provides visibility into historical usage and cost metrics such as run duration, credit usage and failures. This feature is available as a top level tab in the Dagster Cloud UI:

<Image
alt="Viewing the Insights tab in the Dagster UI"
src="/images/dagster-cloud/insights/insights-tab.png"
width={771}
height={536}
/>

> The Insights page shows a list of metrics in the left panel. For each metric the daily, weekly or monthly aggregated values are shown in a graph in the main panel. As of October 2023 the metrics are update once a day. The following metrics are available:

Reviewer: ```suggestion
The Insights page shows a list of metrics in the left panel. For each metric the daily, weekly or monthly aggregated values are shown in a graph in the main panel. These metrics are updated available on a daily cadence. The following metrics are available:
```

phrasing nit/suggestion

> - **Dagster credits**: Dagster credit cost associated with computing this object. Dagster credits are charged for every step that is run, and for every asset that is materialized.

Reviewer: we should link to the pricing page here
```suggestion
- **Dagster credits**: Dagster credit cost associated with computing this object. Dagster credits are charged for every step that is run, and for every asset that is materialized. For more information, [see the pricing FAQ](https://dagster.io/pricing#faq).
```

- **Compute duration**: Time spent computing steps. For jobs which run steps in parallel, the compute duration may be longer than the wall clock time it takes for the run to complete.

> - **Retry compute**: "Time spent computing steps, including time spent retrying failed steps. For jobs which run steps in parallel, the compute duration may be longer than the wall clock time it takes for the run to complete.

Reviewer: ```suggestion
- **Retry compute**: Time spent computing steps, including time spent retrying failed steps. For jobs which run steps in parallel, the compute duration may be longer than the wall clock time it takes for the run to complete.
```
